### DISCUSSION TITLE
Probe not deploying/probe out of bounds

### DISCUSSION THREAD

**Discussion by 90bookwyrm** [mriscoc/Ender3V2S1#1287] [Q&A]
I am using the ender3v2-427-bltubl-mpc-20240125.bin firmware on and ender 3v2 with a bl touch and a 4.2.7 motherboard. When I started the printer the first time, the probe deployed on startup. I then ran the auto framing wizard and it ran as expected through the first 6 probes. Then threw a probe out of bounds error. 

Now the probe does not deploy at all on startup, teaming, or on the deploy command. It just blinks red. I have reset all the settings, opened the bl touch and confirmed the pin is straight and moving well. What happened and how do I fix it?

**Comment by mriscoc** [comment]
Check mesh inset https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides, try disabling HS mode and Bltouch wiring.
